mixer/pkg/api/grpcServer_test.go

```diff
@@ -596,6 +604,40 @@ func TestFailingPreproc(t *testing.T) {
 	}
 }
 
+func TestOutOfSyncDict(t *testing.T) {
+	ts, err := prepTestState()
+	if err != nil {
+		t.Fatalf("Unable to prep test state: %v", err)
+	}
+	defer ts.cleanupTestState()
+
+	{
+		request := mixerpb.CheckRequest{GlobalWordCount: 9999}
+		resp, err := ts.client.Check(context.Background(), &request)
+		if resp != nil {
+			t.Error("Expecting no response, got one")
+		}
+		if err == nil {
+			t.Error("Got success, expected failure")
+		} else if !strings.Contains(err.Error(), "inconsistent dictionary versions") {
```

Comment:
can we avoid using hardcoded error string checks here somehow?  like make `DictionaryError` a type that we can detect?

Reply:
Not so easy since this is going through gRPC which turns everything into a generic status.Error value. I tried passing an explicit type and testing for that, but the type is erased. The only thing preserved is the string.